osd_cluster_create/managing-byo-vpc-single-az.adoc

```diff
@@ -0,0 +1,29 @@
+[id="managing-byo-vpc-single-az"]
+= Managing your own virtual private cloud with a single availability zone in {product-title}
+:context: byo-vpc-single-az
+include::modules/attributes-openshift-dedicated.adoc[]
+
+toc::[]
+
+Configuring a cluster to include your own pre-configured virtual private cloud (VPC) allows for the use of any existing network that relies on an existing VPC.
+
+[id="byo-vpc-single-preq"]
+== Prerequisites
+
+* You have a configured VPC.
+* You have a configured AWS account in your AWS organization. The account should have the following:
+.. an IAM user with the name `osdCcsAdmin`.
+.. The `AdministratorAccess` policy attached to this role.
```

Comment:
```suggestion
.. An IAM user with the name `osdCcsAdmin`
.. The `AdministratorAccess` policy attached to this role
```
see above

Reply:
When you say `role` here, do you mean this `IAM user`? If so, consider making this explicit to avoid confusion.